superset/key_value/commands/create.py

```diff
@@ -33,14 +33,14 @@ class CreateKeyValueCommand(BaseCommand, ABC):
     def __init__(
         self, user: User, resource_id: int, value: str,
     ):
-        self._actor = user
+        self._user = user
```

Comment:
I think "actor" is perhaps more descriptive here:
- the "actor" is doing something
- the key-value pair has an "owner"